src/libs/ReportActionsUtils.ts

```diff
@@ -1348,6 +1350,34 @@ function isLinkedTransactionHeld(reportActionID: string, reportID: string): bool
     return TransactionUtils.isOnHoldByTransactionID(getLinkedTransactionID(reportActionID, reportID) ?? '-1');
 }
 
+function getAccountIDsFromMessage(message: string) {
+    const regex = /<mention-user\s+accountID="(\d+)"\/>/g;
+    const matches = [];
+    let match;
+    // eslint-disable-next-line no-cond-assign
+    while ((match = regex.exec(message)) !== null) {
+        matches.push(match[1]);
+    }
+    return matches;
+}
+
+function getMentionedEmailsFromMessage(message: string) {
+    const regex = /<mention-user>(.*?)<\/mention-user>/g;
+    const matches = [];
+    let match;
+    // eslint-disable-next-line no-cond-assign
+    while ((match = regex.exec(message)) !== null) {
+        matches.push(Str.removeSMSDomain(match[1].substring(1)));
+    }
+    return matches;
+}
+
+function didMessageMentionCurrentUser(message: string) {
+    const accountIDsFromMessage = getAccountIDsFromMessage(message);
+    const emailsFromMessage = getMentionedEmailsFromMessage(message);
+    return accountIDsFromMessage.includes(String(currentUserAccountID)) || emailsFromMessage.includes(currentEmail);
```

Comment:
We also need to include `@here` mentions, since that's also a mention of the user.